saltwater-repl/src/main.rs

```diff
@@ -0,0 +1,17 @@
+//! The repl implementation for saltwater.
+#![deny(rust_2018_idioms)]
```

Comment:
This is such a badly named lint 😆

Reply:
Should I remove it?

Reply:
No no, you're fine, just that `deny(idioms)` looks weird.

Reply:
Yea that's true